docs/configure.md

```diff
@@ -157,6 +157,74 @@ aws s3 ls --profile my-sso-profile
 `aws-sso-util configure populate` allows you to configure profiles for all the access you have through AWS SSO.
 You specify one or more regions, and a profile is created for every account, role, and region you have access to through AWS SSO (note that if access to a region is prohibited by an IAM policy, this does not suppress creation of the profile).
 
+## Options
```

Comment:
Pull this section out. It's a good point that documentation of the options is missing but I'd rather have it formatted the same as other commands. I can do that after.